ooni/nettests/blocking/web_connectivity.py

```diff
@@ -77,6 +77,12 @@ class WebConnectivityTest(httpt.HTTPTest, dnst.DNSTest):
     requiresTor = False
     followRedirects = True
 
+    # These are the options to be shown on the GUI
+    simpleOptions = [
+        {"name": "url", "type": "text"},
```

Comment:
Because this is actually not a file, but a single URL to be put inside of an `<input type='text'>`.